code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -282,6 +282,82 @@
 	job = list("Head of Personnel", "Quartermaster", "Cargo Technician", "Librarian", "Coroner", "Psychiatrist", "Virologist")
 
 
+//--------------------------//
+// Species Restricted Gear //
+//-------------------------//
+
+/datum/uplink_item/species_restricted
+	category = "Species Specific Gear"
+	cant_discount = TRUE
+	excludefrom = list(UPLINK_TYPE_NUCLEAR, UPLINK_TYPE_SST) // Stops the job specific category appearing for nukies
+
+//human
+//NONE! Add me!
+
+//Tajaran
+//TO-DO: retrectable claws
+
+//Unathi
+//TO-DO: Dueling Energy Sword
+
+//skrell
+/datum/uplink_item/species_restricted/lovepen
+	name = "Aggression Supression Pen"
+	desc = "A syringe disguised as a functional pen. It's filled with a potent aggression supressant. \ The pen holds four doses of the mixture. The pen can be refilled "
+	reference = "LP"
+	item = /obj/item/pen/love
+	cost = 4
+	species = list("Skrell")
+
+//Slime people
+// no idea
```

Comment:
I appreciate seeing what might be coming down the pipeline, but we don't necessarily need them here in code, especially when they're things like "no idea".